### PR TITLE
Add support for merged plots.

### PR DESCRIPTION
**Pull request by achoum**
Event-set passed in tuples are plotted together.

Example:

```
tp.plot([
evset_1, # Plot each feature of “evset_1” individually
(evset_1,), # Plot all the features of “evset_1” together
(evset_1[["a", "b"]], evset_2[“c”]), # Plot “a” and “b” from “evset_1”, and “c” from evset_2 together.
])
```

### CODE REVIEW

File: temporian/implementation/numpy/data/plotter.py [hunk 1]
```diff
@@ -14,53 +14,206 @@
...
+            groups.append(Group([GroupItem(evsets, -1)]))
+        return groups
+
+    if isinstance(evsets, tuple):
+        # Plot all the event sets and their features together
+        group_items = []
+        for evset in evsets:
+            if not isinstance(evset, EventSet):
+                raise ValueError(
+                    f"Expecting tuple of eventsets. Got {type(evset)} instead."
```
Comment (DonBraulio) [inline temporian/implementation/numpy/data/plotter.py:137]: nit: `tuple of EventSets`

File: temporian/implementation/numpy/data/plotter.py [hunk 2]
```diff
@@ -14,53 +14,206 @@
...
-class Style(Enum):
-    """Plotting style."""
+def normalize_features(features: InputFeatures) -> Optional[Set[str]]:
+    """Normalizes the "features" argument of plot."""
+
+    if features is None:
+        return None
+    if isinstance(features, str):
+        return {features}
+    if isinstance(features, list):
```
Comment (DonBraulio) [inline temporian/implementation/numpy/data/plotter.py:165]: `if isinstance(features, (list, tuple))`

File: temporian/implementation/numpy/data/plotter.py [hunk 3]
```diff
@@ -242,8 +245,12 @@ def plot(
         # Plot each feature individually
         >>> tp.plot(evset)
 
-        # Plots multiple features in the same sub-plot.
-        >>> tp.plot((evset,), interactive=True)
+        # Plots multiple features in the same sub-plot (equivalent).
```
Comment (DonBraulio) [inline temporian/implementation/numpy/data/plotter.py:248]: Remove `(equivalent)` ? I guess it's for the example below

File: temporian/implementation/numpy/data/plotter.py [hunk 4]
```diff
@@ -242,8 +245,12 @@ def plot(
...
         >>> tp.plot(evset)
 
-        # Plots multiple features in the same sub-plot.
-        >>> tp.plot((evset,), interactive=True)
+        # Plots multiple features in the same sub-plot (equivalent).
+        >>> tp.plot(evset, merge=True)
+
+        >>> evset_2 = tp.event_set([5, 6])
+        >>> tp.plot([evset, evset_2], merge=True)
+        >>> tp.plot((evset, evset_2))
```
Comment (DonBraulio) [inline temporian/implementation/numpy/data/plotter.py:253]: Add comment `# equivalent` before these two plots.

**Review by DonBraulio — APPROVED** [review]
Cool 🙌
Didn't fully review in-depth but left some comments.

File: temporian/implementation/numpy/data/plotter.py (hunk 2 above)
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:165]: We are not expecting for the features to be tuples

File: temporian/implementation/numpy/data/plotter.py (hunk 3 above)
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:248]: *Done

File: temporian/implementation/numpy/data/plotter.py (hunk 4 above)
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:253]: *Done

File: temporian/implementation/numpy/data/plotter.py [hunk 5]
```diff
@@ -258,7 +265,11 @@ def plot(
         ```
 
     Args:
-        evsets: Single or list of EventSets to plot.
+        evsets: EventSet, list of EventSets, tuple of EventSets, or list of
+            tuple of EventSets to plot. If feeding individual EventSets and list
+            of EventSets, individual features are plotted in different
+            sub-plots. If feeding tuple of EventSets, all features are plotted
+            together in a same plot.
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:272]: Good point.
*Done

File: temporian/implementation/numpy/data/plotter.py (hunk 1 above)
Comment (achoum) [reply temporian/implementation/numpy/data/plotter.py:137]: *Done

**Comment by achoum** [comment]
Thanks :)

**Pull request merged**